### PR TITLE
feat(integrations): Add assignee sync from sentry to JIRA

### PR DESCRIPTION
**Pull request by macqueen**
screenshot 2018-06-19 12 47 40

### CODE REVIEW

File: src/sentry/integrations/jira/integration.py [hunk 1]
```diff
@@ -407,6 +409,54 @@ def create_issue(self, data, **kwargs):
...
 
+    def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
+        """
+        Propagate a sentry issue's assignee to a jira issue's assignee
+        if user is None, unassign
+        """
+        client = self.get_client()
+
+        jira_user = None
+        if user is not None:
```
[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/integrations/jira/integration.py]: oo good call. i had this branch done before making those changes and forgot to update, but will fix.

File: src/sentry/integrations/jira/integration.py [hunk 2]
```diff
@@ -407,6 +409,54 @@ def create_issue(self, data, **kwargs):
...
+                    jira_user = [
+                        r for r in res if r['emailAddress'] == ue.email
+                    ][0]
+                except IndexError:
+                    pass
+                else:
+                    break
+
+            if jira_user is None:
+                # TODO(jess): do we want to email people about these types of failures?
```
[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/integrations/jira/integration.py]: yeah, filed a ticket for this so one of us can come back to it: https://getsentry.atlassian.net/browse/APP-355.

File: src/sentry/integrations/jira/integration.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (macqueen) [reply src/sentry/integrations/jira/integration.py]: fixed

**Review by lauryndbrown — APPROVED** [review]
Looks good to me

**Pull request merged**